src/models/user.py

```diff
@@ -0,0 +1,25 @@
+from database import db
+from werkzeug.security import generate_password_hash, check_password_hash
+from flask_login import UserMixin
+from flask_login import LoginManager
+
+login = LoginManager()
```

Comment:
I would rather define this and the `@login.user_loader` decorator in a new file `src/login.py`. It also needs to be registered with the app in `src/app.py`, so it is similar to the database.